### PR TITLE
feat: Update borders on the draft detours table

### PR DESCRIPTION
**Pull request by joshlarson**
I also snuck a few other changes in, like fixing the font and background colors, since it looked annoyingly bad without those changes.

# Before

Screenshot 2024-08-28 at 5 52 11 PM

# After

Screenshot 2024-08-28 at 5 52 52 PM

---

Asana Ticket: https://app.asana.com/0/1205385723132845/1208155044119310/f

### CODE REVIEW

**Review by hannahpurcell — CHANGES_REQUESTED** [review]
Agh! Both luckily and unluckily, I use Chrome, and so I spotted a uniquely Chrome issue with the border-radius in all corners, the upper-corners visible in the screenshot. I tried finding a quick and easy answer online to resolve this, and didn't find it in 8 minutes, so I'm afraid there might be a curveball here...

Screenshot 2024-08-29 at 5 14 56 PM

**Comment by joshlarson** [comment]
@hannahpurcell Fascinating! Apparently you're just not supposed to use `border-radius` on tables, only on individual cells. Firefox lets you _sooooort_ of pretend that it works, but Chrome doesn't.

[I think it's fixed now](https://github.com/mbta/skate/pull/2762/commits/876d529b1007d143e1aede00fce904e9f1aee122) if you'd be interested in taking another look - thanks for catching this!

File: assets/css/detours/_detour_table.scss [hunk 1]
```diff
@@ -1,8 +1,65 @@
+@use "../color/tokens_2024" as tokens;
+
+$table-border: 1px solid tokens.$gray-500;
```
Comment (hannahpurcell) [inline assets/css/detours/_detour_table.scss:3]: The `$gray-500` was looking a little intense to my eye, and checking the [figma](https://www.figma.com/design/iQiQA1toiMfPDhSbTYg2cx/Skate-%7C-Unplanned-Detours-%7C-Exploration?node-id=1783-2059&t=qnIFHo7vhooGGzaH-4), it seems the border should a bit more delicate: `#00000020`. (I'm not worried about making it a reusable color.)
```suggestion
$table-border: 1px solid #00000020;
```

Comment (hannahpurcell) [reply assets/css/detours/_detour_table.scss:3]: Actually, the figma I attached was not the final handed off version. I cannot find the handoff doc

Comment (hannahpurcell) [reply assets/css/detours/_detour_table.scss:3]: Ok yep! This is different in the official [hand-off](https://www.figma.com/design/iQiQA1toiMfPDhSbTYg2cx/Skate-%7C-Unplanned-Detours-%7C-Exploration?node-id=1956-2816&t=vAH4efrkwgE3UVA6-4). (How interesting!)

Sadly, the handoff uses 2021 colors 🥲  I'd rather keep this file relying on the 2024 colors, though. It doesn't seem worthwhile using a deprecated color for a border, of all things.

```suggestion
$table-border: 1px solid tokens.$gray-400;
```

Comment (joshlarson) [reply assets/css/detours/_detour_table.scss]: Fascinating - not sure how I goofed that

**Review by hannahpurcell — APPROVED** [review]
Great! 👍

**Pull request merged**